README.md

```diff
@@ -157,7 +157,7 @@ If an array, sort keys in ordering of `options.sortOrder`.
 
 ```js
 const sorted = sortPackageJson(packageJsonObject, {
-  sortOrder: ['version']
+  sortOrder: ['version'],
```

Comment:
Thank god for this! Having to disable prettier and undo the added commas every time I edited the readme was terrible